kref01-sp

FilipKresl opened the pull request:
Pilotní verze semestrální práce. 

Ukázka slq queries v UsersDB.php. Řada dalších souborů je rozpracována víceméně z poloviny, připojení k databázi zatím jen na localhost

FilipKresl committed:
Add files via upload

FilipKresl committed:
verze sp k obhajobe

FilipKresl commented:
Aplikace dostupná na:
https://eso.vse.cz/~kref01/sp/

FilipKresl committed:
Delete www/kref01/sp/classes/DatabaseConnection.php

FilipKresl committed:
nahrani DatabaseConnection

nvbach91 commented:
aplikace neni funkcni

![image](https://github.com/nvbach91/4IZ278-2023-2024-LS/assets/20724910/0087a53f-3ffa-4326-97b0-dd76158ffbdc)

nvbach91 reviewed (changes requested):
app not working

FilipKresl commented:
@nvbach91  omlouvám se, aplikace nyní funkční.

Pro snadnější testování přikládám login credentials na vytvořené test účty.

### Students

john.doe@example.com
mWQJKOph2QGC6DA

jane.smith@example.com
kdGYyD7QSz3KFSO

michael.johnson@example.com
E4MPEp5Rv9tTVPl

emily.doe@example.com
WIBmNeWiqxxT4FX

daniel.doe@example.com
xojvDhZxmyYUN7b

### Teachers

alice.johnson@example.com
9guTFX9ktthVKpA

mark.williams@example.com
aeqzZZXuuSoLCCl

### Parents

robert.doe@example.com
OLvZrmCaWrH32Hb

mary.smith@example.com
ODSSd1h6H9qff69

### Admin

kref01@vse.cz
oYXQrmTvlDotttc

FilipKresl committed:
Delete redundant www/kref01/sp/components/registration.php

aplikace nyní funkčí. Na serveru opraveno staré heslo k db.

nvbach91 commented:
- chybi datum a cas kdy doslo k odevzdani ukolu, navic pridejte osestreni vstupu prazdneho textu vypracovaneho ukolu...
- kdyz jsem zalozil novy assignment, nedostanou ho vsichni, ale jen nekteri - kde si urcim kteri studenti dostanou muj ukol resp. vztah student-urcitel?

FilipKresl committed:
fixed verze na obhajobu

added adminDashboard.php, datum a cas v homework.php, osetreni odevzdani prazdneho ukolu

FilipKresl commented:
Diky @nvbach91 !

> * chybi datum a cas kdy doslo k odevzdani ukolu, navic pridejte osestreni vstupu prazdneho textu vypracovaneho ukolu...

datum a cas jsem pridal do homework.php, prazdne odevzdani osetreno

> * kdyz jsem zalozil novy assignment, nedostanou ho vsichni, ale jen nekteri - kde si urcim kteri studenti dostanou muj ukol resp. vztah student-urcitel?

dostanou ho ti studenti, kteri jsou v danem kurzu. Management kurzu s celou radou dalsich admin funkci pridan do adminDashboard.php (je treba se prihasit jako admin (prihlasovaci udaje viz par komentaru vyse), zobrazi se nahore v navigacnim baru)

nvbach91 commented:
- studenti at nevidi id, a nepotrebne veci
- rodice chteji videt hodnoceni nejnovejsi prvni
- id assignmentu --> nazvem
- samostatna stranka pro jeden kurz, kde bude mozne editovat, pridavat studenty hromadne selektem

FilipKresl committed:
fixed veci behem obhajoby

checkbox pro pridavani useru do kurzu
